logstash-core/lib/logstash/lir_serializer.rb

```diff
@@ -0,0 +1,124 @@
+# encoding: utf-8
```

Comment:
I'd recommend namespacing this class with the other pipeline config-related things:

~~~
LogStash::Config::LIRSerialiser
~~~

and moving it to `logstash-core/lib/logstash/config/lir_serialiser.rb`